### PR TITLE
Autos

### PR DESCRIPTION
**Pull request by Aradig26**

### CODE REVIEW

**Review by JashKotadiya — COMMENTED** [review]
LGTM

**Review by elleryjh — APPROVED** [review]

File: src/main/java/frc/robot/Orchestrator.java [hunk 1]
```diff
@@ -191,6 +204,10 @@ public Command indexBasic() {
         .withTimeout(1);
   }
 
+  public Command stopFlywheel() {
+    return Commands.run(() -> shooter.manualShoot(0));
```
Comment (elleryjh) [inline src/main/java/frc/robot/Orchestrator.java:208]: This does nothing - it runs a method that returns the Command, and doesn't run the command.

It should just be: `return shooter.manualShoot(0);`

Comment (admlvntv) [reply src/main/java/frc/robot/Orchestrator.java:208]: Oops

Comment (Aradig26) [reply src/main/java/frc/robot/Orchestrator.java:208]: true. I was thinking manualShoot was a runnable, didnt realize it was a command

**Review by EQMoore — APPROVED** [review]
LGTM

**Pull request merged**